node/src/components/block_validator/state.rs

```diff
@@ -370,11 +370,13 @@ impl BlockValidationState {
                             "no further missing transactions - block validation complete"
                         );
                         let new_state = BlockValidationState::Valid(appendable_block.timestamp());
+                        println!("OK {:?}", responders);
```

Comment:
Are these prints intentional?

Reply:
i left it there to call attn for Karan...the numbers are questionable